src/modules/CRUD/components/Details/Create/Create.js

```diff
@@ -0,0 +1,186 @@
+import Header from '../Header';
+
+
+const submitFeature = props => async ({ formData: { geom, ...properties } }) => {
+  const {
+    isMounted,
+    view: {
+      featureEndpoint,
+      layer: { name },
+    },
+    history: { push },
+    saveFeature,
+    setLoading,
+    setSchema,
+    t,
+  } = props;
+  setLoading(true);
+
+  setSchema(schema => {
+    const defaultValues = Object.entries(properties);
+    if (defaultValues.length) {
+      const nextSchema = { ...schema };
+      defaultValues.forEach(([key, value]) => {
+        nextSchema.properties[key].default = value;
+      });
+      return nextSchema;
+    }
+    return schema;
+  });
+
+  const savedFeature = await saveFeature(featureEndpoint, false, { geom, properties }, 'POST');
+
+  if (!isMounted.current) return;
+
+  setLoading(false);
+
+  if (savedFeature) {
+    push(generateURI('layer', { layer: name, id: savedFeature.identifier }));
+    toast.displayToaster(
+      { id: savedFeature.identifier },
+      t('CRUD.details.successCreateFeature'),
+    );
+  }
+};
+
+const buildProperties = props => {
+  const propertiesWithoutGroup = Object.values(props).reduce((list, value) => {
+    const { type, properties, required = [] } = value;
+
+    if (type === 'object' && required.length) {
+      const requiredProps = Object.entries(properties).reduce((acc, [subKey, subValue]) => (
+        required.includes(subKey)
+          ? ({ [subKey]: subValue })
+          : acc
+      ), {});
+
+      return {
+        ...list,
+        properties: { ...list.properties, ...requiredProps },
+        required: [...list.required, ...required],
+      };
+    }
+
+    return list;
+  }, { properties: {}, required: [] });
+
+  return propertiesWithoutGroup;
+};
+
+const buildSchema = ({ schemaProperties, uiSchemaProperties, geomType }) => {
+  const {
+    schema: geomSchema,
+    ui_schema: geomUiSchema,
+    value,
+  } = getJSONSchemaFromGeom({ identifier: null, geom: null, title: 'Geométrie', geom_type: geomType });
+
+  const { required, properties } = buildProperties(schemaProperties);
+
+  const { 'ui:order': uiOrder = [] } = uiSchemaProperties;
+
+  return {
+    schema: {
+      properties: { geom: { ...geomSchema, default: value }, ...properties },
+      required: ['geom', ...required],
+      type: 'object',
+    },
+    uiSchema: {
+      geom: geomUiSchema,
+      ...uiSchemaProperties,
+      'ui:order': ['geom', ...uiOrder],
+    },
+  };
+};
+
+const Create = props => {
+  const {
+    view: {
+      formSchema: { properties: schemaProperties } = {},
+      uiSchema: uiSchemaProperties,
+      layer: { geom_type: geomType, name },
+    },
+    t,
+  } = props;
+
+  const isMounted = useRef(true);
+
+  useEffect(() => {
+    isMounted.current = true;
+    return () => { isMounted.current = false; };
+  }, []);
+
+  const title = t('CRUD.details.create', { layer: name });
+
+  const [loading, setLoading] = useState(false);
```

Comment:
I think hooks should be at the top of component, just after or before the `useRef`.